src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -93,46 +99,10 @@ def put(self, request: Request, organization, alert_rule) -> Response:
         )
 
         if serializer.is_valid():
-            if not self._verify_user_has_permission(request, alert_rule):
-                return Response(
-                    {
-                        "detail": [
-                            "You do not have permission to edit this alert rule because you are not a member of the assigned team."
-                        ]
-                    },
-                    status=403,
-                )
             alert_rule = serializer.save()
             return Response(serialize(alert_rule, request.user), status=status.HTTP_200_OK)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
     def delete(self, request: Request, organization, alert_rule) -> Response:
-        if not self._verify_user_has_permission(request, alert_rule):
```

Comment:
@ceorourke we don't need this anymore?

Reply:
https://github.com/getsentry/sentry/pull/53970#discussion_r1281102809